spec/source/conventions.html

```diff
@@ -0,0 +1,107 @@
+<emu-clause id="conventions">
+  <h1>Notational Conventions</h1>
+
+  <p>
+    This standard uses a subset of the notational conventions of the ECMAScript 2015 Language Specification (ECMA-262 6th Edition), as ES2015:
```

Comment:
Cool, then we'll have to undo the ones I've already changed, but that should be trivial